### PR TITLE
replace Landau distribution with GSL call

### LINKED ISSUE #15: cernlib dependency
It would be worth to check if a C/C++ replacement exists for the cernlib functions we use in libamtrack

### PR DESCRIPTION
**Pull request by grzanka**
Related to  #15

### CODE REVIEW

**Comment by grzanka** [comment]
@adudzik @marwin1991 this would allow to get rid of Fortran files at some point. What do you think from the perspective of emscripten ? Will it help ?

In next PR I will provide also C-version of Vavilov methods.

File: include/AT_NumericalRoutines.h [hunk 1]
```diff
@@ -402,5 +403,13 @@ double AT_get_interpolated_y_from_interval( const double left_x,
...
 // TODO implement linear interpolation on logarithmic scale
 
+/**
+ * IDF of Landau distribution
+ * Adapted from gsl_ran_landau by explicitly specifying X variable (from uniform distribution [0,1])
+ * The code is based on CERNLIB G110 method (see http://hep.fi.infn.it/cernlib.pdf)
+ * @param[in] X - random number from uniform distribution [0,1]
+ * @return Landau random number
+ */
+double CL_ranlan_idf(const double X);
```
Comment (marwin1991) [inline include/AT_NumericalRoutines.h]: In forjs branch I've used some file to mock Cernlibfuns, maybe here we should do the same and make some file that will convert gsl_ran_landau and provide funcion that will be used in EnergyLoss.c ?

Comment (marwin1991) [reply include/AT_NumericalRoutines.h]: I mean new file to store function: 
- double CL_ranlan_idf(const double X);

Comment (grzanka) [reply include/AT_NumericalRoutines.h]: I will probably create one when implementing Vavilov distribution.

**Comment by sgreilich** [comment]
I think it is a very good approach to replace the FORTRAN code with C functions, using the former was quite a makeshift solution. Even better if transferring the ROOT code does not generate new dependencies. Regarding the accuracy of the Vavilov implementation in CERNLIB vs. ROOT, I think we should go for the slower, accurate one first, if possible, and check how long it takes to process a mid-SOBP carbon spectum, I would think that is the worst case scenario in CPU load.

File: test/C/AT_test.c [hunk 2]
```diff
@@ -48,6 +48,38 @@
 #include "AT_StoppingPower.h"
 #include "AT_CernlibFuns.h"
 #include "AT_DataRange.h"
+#include "AT_NumericalRoutines.h"
+
+int test_cernlib(){
```
[the remark this answers is not in the document]
Comment (grzanka) [reply test/C/AT_test.c]: It will be done once Vavilov distributions are implemented, meaning next PR

**Pull request merged**